Cohort2/team6/server/src/main/resources/db/migration/V1__ADD_USER.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE users (
+  uuid VARCHAR UNIQUE PRIMARY KEY,
+  email VARCHAR UNIQUE NOT NULL,
+  password VARCHAR NOT NULL,
+  first_name VARCHAR NOT NULL,
+  last_name VARCHAR NOT NULL
+);
```

Comment:
we can modify this later, but this is an example of a query for automated flyway migrations